### PR TITLE
Rollbacks pg14 beta changes for docker publish

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
-FROM postgres:13.4
+FROM postgres:14rc1
```
Comment (onurctirtir) [inline Dockerfile:3]: shouldn't this be 14.0 ?

File: postgres-13/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
```
Comment (onurctirtir) [inline postgres-13/Dockerfile:2]: 1)

```suggestion
# see citusdata/tools/packaging_automation/templates/docker/postgres-13/postgres-13.tmpl.dockerfile.
```

2) Same probably applies to pg-12 file too

3) Also, we should rename pg14 file to pg13 in tools repo as well.

**Review by onurctirtir — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (gurkanindibay) [reply Dockerfile]: You're right. Changed it thanks

File: postgres-13/Dockerfile (hunk 2 above)
Comment (gurkanindibay) [reply postgres-13/Dockerfile]: Fixed it

**Pull request merged**